.github/workflows/node-license-scanning.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install --prod
```

Comment:
@Vivek-NatWest - I'd still keep the `--prod` flag, otherwise you'd end up scanning licenses for dev dependencies, which is not required and can potentially generate a lot of false negatives.

Any specific reason for not using `npm ci --prod` ?

Reply:
I had tried using npm ci --omit=dev , but that causes build fail:
Run npm ci --omit=dev

> @finos/fdc3-backplane-client@0.1.0 prepare
> tsdx build

sh: 1: tsdx: not found
one solution is to include tsdx as dependency.

Reply:
Gotcha. Another solution could be (haven't tested yet) to add a `- run: npm install tsdx` as a build step, before the `npm install --prod`; that should work, and avoid adding `tsdx` as runtime dependency.

Reply:
Thanks!  makes sense, will git it a try.